test/functional/provider/perl_spec.lua

```diff
@@ -19,7 +23,51 @@ before_each(function()
   clear()
 end)
 
-describe('perl host', function()
+describe('legacy perl provider', function()
+  if helpers.pending_win32(pending) then return end
```

Comment:
Which tests fail on Windows?

Reply:
The remote plugin provider doesn't work on Windows currently as its not possible to poll STDIN yet.

Reply:
If or when ```Neovim::Ext``` can use a libuv based backend maybe this would start working. This isn't a new limitation.